### PR TITLE
Core/DB/Updater: engrish corrections in DBUpdater.cpp

### PR DESCRIPTION
**Pull request by ghost**
**Changes proposed**:

English grammar and syntax corrections in DBUpdater.cpp
- string texts corrected, source code unchanged
- comments improved and checked for typos

**Target branch(es)**: 3.3.5 + 6.x 
(the 6.x file contains extra lines for the `hotfixes` DB, but the modified text is shared between the files)

**Issues addressed**: Fixes incorrect English and some imprecise language

**Tests performed**: (Does it build, tested in-game, etc)

Builds without problem. Only text strings and comments have been changed.

**Known issues and TODO list**:
- [X] Wait for feedback from TC devs in case of different interpretations in the text
- [X] Update PR file with changes recommended by developers

### CODE REVIEW

File: src/server/database/Updater/DBUpdater.cpp [hunk 1]
```diff
@@ -280,15 +280,15 @@ bool DBUpdater<T>::Populate(DatabaseWorkerPool<T>& pool)
...
+                TC_LOG_ERROR("sql.updates", ">> Base file \"%s\" is missing. Try fixing it by cloning the source again.",
                     base.generic_string().c_str());
 
                 break;
             }
             case LOCATION_DOWNLOAD:
             {
                 TC_LOG_ERROR("sql.updates", ">> File \"%s\" is missing, download it from \"https://github.com/TrinityCore/TrinityCore/releases\"" \
-                    " and place it in your server directory.", base.filename().generic_string().c_str());
+                    " and place it in your core server directory.", base.filename().generic_string().c_str());
```
Comment (Naios) [inline src/server/database/Updater/DBUpdater.cpp]: worldserver not core server

Comment (tkrokli) [inline src/server/database/Updater/DBUpdater.cpp]: Thanks, I'll fix that too. :+1:

File: src/server/database/Updater/DBUpdater.cpp [hunk 2]
```diff
@@ -178,7 +178,7 @@ bool DBUpdater<T>::Create(DatabaseWorkerPool<T>& pool)
     // Path of temp file
     static Path const temp("create_table.sql");
 
-    // Create temporary query to use external mysql cli
+    // Create temporary query to use external MySQL CLi
```
Comment (Rochet2) [inline src/server/database/Updater/DBUpdater.cpp]: Hmm, shouldnt the **i** be capital as well as it is short for command line interface, no?

Comment (tkrokli) [inline src/server/database/Updater/DBUpdater.cpp]: Fair point, I just read it as "Command Line interface" when I see it spelled `CLi` elsewhere. :-)
It's a moot point now that this PR is merged. I wasn't quick enough to read your comment. ;)

**Pull request merged**